content/references/protocol-reference/ledger-data/ledger-object-types/offer.ja.md

```diff
@@ -41,40 +41,40 @@ labels:
+|-------------------|-----------|-----------|------|-------|
+| `Account`           | 文字列    | AccountID | はい  | このオファーを所有するアカウントのアドレス。 |
+| `BookDirectory`     | 文字列    | Hash256   | はい  | このオファーにリンクしている[オファーディレクトリー](directorynode.html)のID。 |
+| `BookNode`          | 文字列    | UInt64    | はい  | Offerディレクトリが複数ページで構成されている場合に、このオブジェクトにリンクしているページを示すヒント。 |
+| `Expiration`        | 数値    | UInt32    | いいえ | （省略可）このオファーが資金不足とみなされる時刻。詳細は、[時間の指定][]を参照してください。 |
+| `Flags`             | 数値    | UInt32    | はい  | このオファーに対して有効になっているブール値フラグのビットマップ。 |
+| `LedgerEntryType`   | 文字列    | UInt16    | はい  | 値が`0x006F`（文字列`Offer`にマッピング）の場合は、このオブジェクトが通貨取引オーダーを記述することを示す。 |
+| `OwnerNode`         | 文字列    | UInt64    | はい  | 所有者ディレクトリーが複数ページで構成されている場合に、このオブジェクトにリンクしているページを示すヒント。**注記:** このオファーには、オファーを含む所有者ディレクトリーへの直接リンクは含まれていません。これは、その値を`Account`から取得できるためです。 |
+| `PreviousTxnID`     | 文字列 | Hash256 | はい  | 最後にこのオブジェクトを変更したトランザクションの識別用ハッシュ。 |
+| `Sequence`          | 数値    | UInt32    | はい  | `Offer`オブジェクトを作成した[OfferCreate][]トランザクションの`Sequence`値。`Account`とこのフィールドの組み合わせによってこのオファーが識別されます。 |
+| `PreviousTxnLgrSeq` | 数値 | UInt32 | はい  | 最後にこのオブジェクトを変更したトランザクションが記録された[レジャーインデックス][]。 |
+| `TakerPays`         | 文字列またはオブジェクト | Amount | はい  | オファー作成者が要求する残額と通貨の種類。 |
+| `TakerGets`         | 文字列またはオブジェクト | Amount | はい  | オファー作成者が提供する残額と通貨の種類。 |
 
 ## Offerのフラグ
 
-[OfferCreateトランザクション][]でOfferオブジェクトを作成するときに有効化または無効化できる各種オプションがあります。レジャーではフラグはバイナリ値として表され、これらのバイナリ値はビットOR演算と組み合わせることができます。レジャーでのフラグのビット値は、トランザクションでこれらのフラグを有効または無効にするために使用する値とは異なります。レジャーのフラグには、 _lsf_ で始まる名前が付いています。
+[OfferCreateトランザクション][]でOfferオブジェクトを作成するときに有効化または無効化できる各種オプションがあります。レジャーではフラグはバイナリ値として表され、これらのバイナリ値はビットOR演算と組み合わせることができます。レジャーでのフラグのビット値は、トランザクションでこれらのフラグを有効または無効にするために使用する値とは異なります。レジャーのフラグには、 **`lsf`** で始まる名前が付いています。
 
-`Offer` オブジェクトには以下のフラグ値を指定できます。
+`Offer`オブジェクトには以下のフラグ値を指定できます。
 
-| フラグ名 | 16進値 | 10進値 | 説明 | 対応する[OfferCreateフラグ](offercreate.html#offercreateフラグ) |
+| フラグ名 | 16進数値 | 10進数値 | 対応する[OfferCreateフラグ](offercreate.html#offercreateフラグ) | 説明 |
 |-----------|-----------|---------------|-------------|------------------------|
-| lsfPassive | 0x00010000 | 65536 | オブジェクトはパッシブオファーとして配置されました。レジャー内のオブジェクトには影響しません。 | tfPassive |
-| lsfSell   | 0x00020000 | 131072 | オブジェクトはセルオファーとして配置されました。レジャー内のオブジェクトへの影響はありません。これは、要求したレートよりも良いレートを得た場合にのみtfSellが関連するためです。この状況はオブジェクトがレジャーに記録された後では発生することはありません。 | tfSell |
+| lsfPassive | `0x00010000` | 65536 | tfPassive | オブジェクトはパッシブオファーとして発注されています。レジャー内のオブジェクトには影響しません。 |
+| lsfSell   | `0x00020000` | 131072 | tfSell | オブジェクトは売却オファーとして発注されています。これは台帳にあるオブジェクトには何の影響もありません (`tfSell`は指定したレートよりも良いレートが存在する場合にのみ意味を持ち、台帳にこのフラグを持ったオブジェクトが入ることはありません。)。 |
 
 ## オファーIDのフォーマット
 
 `Offer`オブジェクトのIDは、以下の値がこの順序で連結されている[SHA-512ハーフ][]です。
 
 * Offerスペースキー（`0x006F`）
 * オファーを行うアカウントのAccountID
-* オファーを作成した[OfferCreateトランザクション][]のシーケンス番号。
-
+* オファーを作成した[OfferCreateトランザクション][]のシーケンス番号
+    OfferCreateトランザクションが[Ticket](tickets.html)を使用した場合、代わりに`TicketSequence`値を使用します。
```

Comment:
This is a formatting error from the English version (addressed in #1760).

It should be like this:

```suggestion
* オファーを作成した[OfferCreateトランザクション][]のシーケンス番号

    OfferCreateトランザクションが[Ticket](tickets.html)を使用した場合、代わりに`TicketSequence`値を使用します。
```

The extra blank line is necessary to force a line break between the sections